### PR TITLE
feat: updated XCM configuration including Zombienet setup with enabled dev accounts on AssetHub

### PR DESCRIPTION
**Pull request by chexware**
### Summary by OpenAI

**New Feature:**
- Updated XCM configuration in `xbi_config.rs` for enhanced interoperability.
- Enabled dev accounts on AssetHub, providing developers with more flexibility and control.
- Improved Zombienet setup in `zombienet.sh`, enhancing the testing environment.
- Added calls to `build_polkadot`, `build_asset_hub`, and `build_collator` functions in the `spawn()` function, streamlining the process of spawning the zombienet.

> 🎉 With configs anew, and dev accounts too,  
> The chains interlink, in a networked zoo.  
> Zombienet spawns, with a build so fine,  
> In this world of code, your light will shine! 🚀

### CODE REVIEW

File: tests/zombienet/zombienet.sh [hunk 1]
```diff
@@ -198,9 +198,11 @@ confirm_sfx() {
 }
 
 spawn() {
+    build_polkadot
     build_asset_hub
+    build_collator
     echo "Spawning zombienet using provider: $provider..."
-    zombienet --provider="$provider" spawn ./zombienet-xcm.toml
+    zombienet --provider="$provider" spawn ./tests/zombienet/zombienet-xcm.toml
```
Comment (3h4x) [inline tests/zombienet/zombienet.sh:205]: this depends from where do you run the script. I would leave it as is and not change paths.

**Review by 3h4x — CHANGES_REQUESTED** [review]
Looks good to me. Please resolve one issue found and 👍

File: tests/zombienet/zombienet.sh (hunk 1 above)
Comment (chexware) [reply tests/zombienet/zombienet.sh]: Just reverted the paths change

**Review by MaciejBaj — APPROVED** [review]

**Review by 3h4x — APPROVED** [review]
👍

**Pull request merged**